ddlambda.go

```diff
@@ -42,54 +43,65 @@ type (
 		// Site is the host to send metrics to. If empty, this value is read from the 'DD_SITE' environment variable, or if that is empty
 		// will default to 'datadoghq.com'.
 		Site string
-
 		// DebugLogging will turn on extended debug logging.
 		DebugLogging bool
 		// EnhancedMetrics enables the reporting of enhanced metrics under `aws.lambda.enhanced*` and adds enhanced metric tags
 		EnhancedMetrics bool
+		// DDTraceEnabled enables the Datadog tracer.
+		DDTraceEnabled bool
+		// MergeXrayTraces will cause Datadog traces to be merged with traces from AWS X-Ray.
+		MergeXrayTraces bool
 	}
 )
 
 const (
-	// DatadogAPIKeyEnvVar is the environment variable that will be used as an API key by default
+	// The following constants specify the environment variables that will...
+	// ...set the API key
 	DatadogAPIKeyEnvVar = "DD_API_KEY"
-	// DatadogKMSAPIKeyEnvVar is the environment variable that will be sent to KMS for decryption, then used as an API key.
+	// ...be sent to KMS for decryption, then used as an API key.
```

Comment:
nit: It's actually idiomatic in golang to write the full exported name in a comment before it, (it's also usually required by the standard linting tools). I haven't seen the '...' notation used before, so I'm not sure if it's idiomatic or not.

Reply:
Yeah I was not thinking about how this would be surfaced in documentation -- I'll put this back to the way it was